Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALChannelCalibrator.h

```diff
@@ -102,16 +116,15 @@ void EMCALChannelCalibrator<DataInput, DataOutput, HistContainer>::initOutput()
 template <typename DataInput, typename DataOutput, typename HistContainer>
 bool EMCALChannelCalibrator<DataInput, DataOutput, HistContainer>::hasEnoughData(const o2::calibration::TimeSlot<DataInput>& slot) const
 {
-
   const DataInput* c = slot.getContainer();
-  LOG(info) << "Checking statistics";
```

Comment:
Needs to be LOG(DEBUG), otherwise you will create a message flood far beyond what EMCAL is already doing now :)